api/envoy/extensions/filters/listener/original_dst/v3/original_dst.proto

```diff
@@ -17,4 +17,17 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 message OriginalDst {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.config.filter.listener.original_dst.v2.OriginalDst";
+
+  enum Method {
+    // Use sockopt SO_ORIGINAL_DST or IP6T_SO_ORIGINAL_DST to recover the original local address from the DNAT or
+    // REDIRECT iptables target. This is the default method.
+    DNAT = 0;
```

Comment:
Protobuf best practices call for the 0th enum value being reserved for DEFAULT or NOT_SET.  Would you consider adding a separate enum value for DEFAULT?

Reply:
In this case, we're basically saying "client-specific default, set explicit DNAT if that is what you really want". This is consistent with API guidelines for multi-client.

Reply:
Yes, that is what I was saying.